staging/2021/benchmarks/benchmarks.csv

```diff
@@ -0,0 +1,634 @@
+Table 2: Historical MIPS Quality Measure Benchmark Results; created using PY2018 data and PY2020 Eligibility Rules,,,,,,,,,,,,,,,,,
```

Comment:
Can you update this to reflect the change to PY2019 and PY2021?

`Table 2: Historical MIPS Quality Measure Benchmark Results; created using PY2019 data and PY2021 Eligibility Rules,,,,,,,,,,,,,,,,, Measure_Name,Measure_ID,Collection_Type,Measure_Type,Benchmark,Standard_Deviation,Average,Decile_3,Decile_4,Decile_5,Decile_6,Decile_7,Decile_8,Decile_9,Decile_10,Topped_Out,Seven_Point_Cap,High_Priority`

Reply:
Otherwise, the files are within the 2021 directory and look good to me